vignettes/DirichletMultinomial.Rmd

```diff
@@ -0,0 +1,320 @@
+---
+title: "DirichletMultinomial for Clustering and
+        Classification of Microbiome Data"
+author:
+- name: "Khadijah Amusat"
+  affiliation: "Vignette translation from Sweave to R Markdown / HTML"
+date: "`r format(Sys.time(), '%B %d , %Y')`"
+output:
+    BiocStyle::html_document:
+    number_sections: true
+    toc: true
+    toc_depth: 4
+bibliography: references.bib
+link-citations: TRUE
+package: DirichletMultinomial    
+vignette: >
+  %\VignetteIndexEntry{DirichletMultinomial for Clustering and
+                       Classification of Microbiome Data}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r style, echo = FALSE, results = 'asis'}
+  BiocStyle::markdown()
+```
+
+This document illustrates the main features of the `r
+Biocpkg("DirichletMultinomial")`, and in the process replicates key tables and
```

Comment:
Be careful with inline code; if it's split on 2 lines, it will just be printed as text in the knitted file. The following must be on one line:

    `r Biocpkg("DirichletMultinomial")`